addons/l10n_it/data/account.tax.group.csv

```diff
@@ -1,11 +1,11 @@
-id,name,country_id/id
-tax_group_iva_2,IVA 2%,base.it
-tax_group_iva_4,IVA 4%,base.it
-tax_group_iva_5,IVA 5%,base.it
-tax_group_iva_10,IVA 10%,base.it
-tax_group_iva_12,IVA 12%,base.it
-tax_group_iva_21,IVA 21%,base.it
-tax_group_iva_20,IVA 20%,base.it
-tax_group_iva_22,IVA 22%,base.it
-tax_group_imp_esc_art_15,Imponibile Escluso Art.15,base.it
-tax_group_fuori,Fuori Campo IVA,base.it
+id,name,country_id/id,preceding_subtotal
```

Comment:
This can be moved to l10n_it_edi_withholding and that way we can as well give the translations.  The tax group xml can be transformed to csv and we can add it as it is here in English and that way it should appear in the translations of l10n_it_edi_withholding.  (or just put it in Italian, but do not translate the preceding_subtotal)

Reply:
I've put it in Italian.